spec/models/apple_music_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "AppelMusicのmodelテスト", type: :model do
+  let(:apple_music) { AppleMusic.new }
```

Comment:
```suggestion
describe AppleMusic do
```

として`described_class`を使った方が端的に表現できるかも。

Reply:
`described_class`の書き方、初めて知りました👀
こちら、 5e2fb62 にて修正しました💪

また、specの構造におけるトップレベルがわかりづらかったので、トップレベルには`RSpec`の記述を追加しました。